src/main.js

```diff
@@ -1,11 +1,13 @@
 import toGovspeak from './to-govspeak'
 import insertTextAtCursor from 'insert-text-at-cursor'
+import htmlFromHiddenElement from './html-from-hidden-element'
 
 function htmlFromPasteEvent (event) {
-  if (!event.clipboardData) {
-    return
+  if (event.clipboardData) {
```

Comment:
It's great we can support this, but I think the "what's it for" aspect has got lost a little - my first question here was "why is there a hidden element"? Although it's documented in the test, I think we could use a couple of question methods, and perhaps a better name for that import, to better explain what's going on to the read.

Reply:
Fair point, we had more comments before and I think they slipped - can imagine it's a bit voodoo.